src/Board.js

```diff
@@ -1,25 +1,45 @@
 import React, { useState } from "react";
 import Square from "./Square";
-import { TREASURES } from './cards';
+import { TREASURES } from "./cards";
 
 export default function Board() {
-  const randomTreasure = TREASURES[Math.floor(Math.random() * TREASURES.length)].grid;
+  const randomTreasure =
+    TREASURES[Math.floor(Math.random() * TREASURES.length)].grid;
   const [gameBoard, setGameboard] = useState(randomTreasure);
+  const [turn, setTurn] = useState([
+    [1, 1, 1, 1],
+    [1, 1, 1, 1],
+    [1, 1, 1, 1],
+    [1, 1, 1, 1],
+  ]);
 
   function handleClick(r, c) {
     let squares = gameBoard.slice();
+    let turnBoard = turn.slice();
     if (squares[r][c] !== 1) {
       squares[r][c] = "X";
+      turnBoard[r][c] = "0";
```

Comment:
Any reason why you made this a string instead of a number? for consistency, I think we should use numbers here
```suggestion
      turnBoard[r][c] = 1;
```